### ISSUE TITLE
Make README less misleading and include vital information

### ISSUE THREAD

**Issue by Timwi** [abus-aikorea/voice-pro#17]
**Describe the bug**

The README misleads readers into thinking they can download, run and freely use the software.

In reality, all you get is a “TRIAL” version that is limited to just 30-minute videos.

**Expected behavior**

The README should make clear that this is a for-pay commercial product, not free software, and has arbitrarily-imposed limitations.

Make sure to also include any other anti-features I may not have noticed, e.g. telemetry and other spyware.

**Comment by abus-aikorea** [comment]
I agree with your point. This repository includes a free trial of voice-pro. 
However, I would like to clarify one misunderstanding. 

Voice-pro can handle videos of any length, whether it's 30 minutes, 1 hour, or 2 hours. The 30-minute limit means that after that time, you can no longer click the Action button in the WebUI, but it does not mean that the processing task will stop. The results of the tasks are all saved in the workspace folder. Therefore, even if you encounter the 30-minute usage limit, you can simply close the Windows command prompt and run start.bat again. 
In other words, voice-pro can be used indefinitely if you wish. 
Of course, the 30-minute limit might be a bit annoying, and I admit that.

And one more thing... 
Voice-pro is executed in a virtual environment (venv) created using Miniconda, independent of the Windows OS. It does not damage the Windows OS. If you have concerns or doubts about telemetry or spyware, there are countless software options available for detection. Give it a try.

Thank you for your opinion. Have a great day~~
If you need any more help, feel free to ask!

**Comment by Timwi** [comment]
Thank you for your response. I’m surprised by it, and also a little confused. It sounds like you’re happy to let users use the software on longer videos, but if that is so, then why is there a limitation at all? If the answer is “because we want to sell stuff and make money”, then this is still not free software.

**Comment by abus-aikorea** [comment]
Well... I still don't fully understand this part. I hope someone can give me some advice. 
Thank you for using Voice-Pro, and I'm glad if this has been helpful to you.
If you have any more requests, feel free to let me know!

**Comment by lyra-white** [comment]
@abus-aikorea 
I am now confused reading all this. Is this a free software or not? If not, I think MIT license is misleading. 
Is your value add doing the gradle ui? I see there is another issue for docker file. Is that gonna require a trial as well?
